src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -178,10 +173,10 @@ public function overload(string $path, string ...$extraPaths): void
     public function populate(array $values, bool $overrideExistingVars = false): void
     {
         $updateLoadedVars = false;
-        $loadedVars = array_flip(explode(',', $_SERVER['SYMFONY_DOTENV_VARS'] ?? $_ENV['SYMFONY_DOTENV_VARS'] ?? ''));
+        $loadedVars = array_flip(explode(',', (string) ($_SERVER['SYMFONY_DOTENV_VARS'] ?? $_ENV['SYMFONY_DOTENV_VARS'] ?? '')));
```

Comment:
none of added casts to string are needed

Reply:
Extra cas removed